### ISSUE TITLE
Actions from super class aren't being registered

### ISSUE THREAD

**Issue by mikazp** [ktsn/vuex-smart-module#12]
Here is a simple use case

```typescript
import {Actions as SmartActions, Module} from "vuex-smart-module";

abstract class Actions extends SmartActions {
    // this action won't be registered
    fetchItems() {
        // fetch code
    }
}

class ConcreteActions extends Actions {
    deleteItems() {
        
    }
}

export const someModule = new Module({
    actions: ConcreteActions
})
```

`[vuex] unknown action type: someModule/fetchItems`

**Comment by ilajosmanov** [comment]
Do you register this module in root module?

**Comment by mikazp** [comment]
sure,  `deleteItems()` works as expected

**Comment by ktsn** [comment]
Thank you for reporting this!
Fixed in v0.2.4

**Comment by mikazp** [comment]
Nice, thank you for fast reaction and for this wonderful package!
